src/backend/src/services/users.services.ts

```diff
@@ -585,6 +587,22 @@ export default class UsersService {
     return requestedUser.unreadNotifications.map(notificationTransformer);
   }
 
+  /**
+   * Gets all of a user's unread announcements
+   * @param userId id of user to get unread announcements from
+   * @param organization the user's orgainzation
+   * @returns the unread announcements of the user
+   */
+  static async getUserUnreadAnnouncements(userId: string, organization: Organization) {
+    const requestedUser = await prisma.user.findUnique({
+      where: { userId },
+      include: { unreadAnnouncements: getAnnouncementQueryArgs(organization.organizationId) }
+    });
+    if (!requestedUser) throw new NotFoundException('User', userId);
```

Comment:
if were gonna do it like this we need to add a check that the user is in this organization